src/main.py

```diff
@@ -37,54 +52,72 @@ def totp(gen):
 	})
 	# Enable the network connectivity of the browser
 	driver.execute_cdp_cmd('Network.enable', {})
-	# Go to the discord login page
-	driver.get("https://www.discord.com/login")
-	options = webdriver.ChromeOptions() 
-	# Clean up the program log
-	options.add_experimental_option("excludeSwitches", ["enable-logging"])
+	# Go to the discord page
+	driver.get(dispage)
 	# Wait 1 second before typing the email and password
 	time.sleep(1)
+	match mode:

```

Comment:
Related: My earlier comment about refactoring the program based on functions for modes.

Reply:
I didn't see these before I pushed my code so that's on me sorry about that.